### ISSUE TITLE
Axios is forced to validate SSL certificate

### ISSUE THREAD

**Issue by EtienneT** [rangav/thunder-client-support#1124]
**Describe the bug**
Before 2.6.0, we could use axios in a custom filter pre-run script without any problems.  We could configure it to ignore errors about SSL certificates by using `rejectUnauthorized: false` in the httpsAgent:

```
axios.create({
        baseURL,
        httpsAgent: new https.Agent({
            rejectUnauthorized: false
        })
    });
```

This is particularly useful in development.  We use this to test our anti-forgery tokens locally before doing a post that requires validating the anti-forgery token.

Since 2.6.0, even if rejectUnauthorized is set to false, it throws the error:

`unable to verify the first certificate`

**To Reproduce**
Use a local web server setuped with a self-signed certificate for developement purposes.  Then in a custom filter pre-run script, use the following code:

```
const https = require('https');
const axios = require('axios');

// Change the baseURL to your local web server
const client = axios.create({
    baseURL,
    httpsAgent: new https.Agent({
        rejectUnauthorized: false
    })
});

// Change this url
client.get('account/logon');
```

Notice that it will throw "unable to verify the first certificate"

**Expected behavior**
The code doesn't throw any error in 2.5.3.  The bug started happening only in 2.6.0.

**Platform:** 
 - OS: Windows 11
 - vscode version: 1.77.3
 - extension version: 2.6.2

**Your Team Size Using TC:**
10

**Comment by rangav** [comment]
Looks like Axios bug, they removed this option in latest version.

https://github.com/axios/axios/issues/5407

**Comment by EtienneT** [comment]
Thanks for finding this.  There doesn't seem to be any workaround in axios for now from what I can see.

Any way I could use "got" instead of axios in my pre-filter script?  I see that this is the default http library you use for requests in Thunderclient, but when I try to load it either with require or tc.loadModule (even if it is installed with npm install), it always returns undefined.

**Comment by EtienneT** [comment]
One user is suggesting a workaround in axios:
https://github.com/axios/axios/issues/5407#issuecomment-1516190693

But I can't access tls in my pre-run script.

Is there an official list of supported packages we can use in prerun scripts?

Thanks,

**Comment by rangav** [comment]
This Bug is fixed now, please update to `v2.7.0`

See all features released
https://github.com/rangav/thunder-client-support/releases/tag/v2.7.0

Please let me know your feedback.
